app/models/mixins/child_storage_manager_mixin.rb

```diff
@@ -0,0 +1,5 @@
+module ChildNetworkManagerMixin
```

Comment:
```suggestion
module ChildStorageManagerMixin
```

not sure if this is clobbering the other module on you.

Reply:
Gah yeah copy&paste error, don't think that's what was doing it though